### PR TITLE
For #15720 - Remove unused telemetry

### PR DESCRIPTION
**Pull request by codrut-topliceanu**
For https://github.com/mozilla-mobile/fenix/issues/15720
### Pull Request checklist

- [x] **Tests**: This PR doesn't include any tests as it is only a minor change.
- [x] **Screenshots**: This PR has no user visible changes.
- [x] **Accessibility**: The code in this PR  does not include any user facing features.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/metrics.yaml [hunk 1]
```diff
@@ -1625,67 +1610,6 @@ activation:
     no_lint:
       - USER_LIFETIME_EXPIRATION
 
-qr_scanner:
```
Comment (liuche) [inline app/metrics.yaml:1628]: cc @eliserichards for qr scanner telemetry - when chatting with vesta we had the question about whether we ported the original QR telemetry over to the new search experience. If it's not relevant to the new experience then we wanted to remove it.

Comment (liuche) [reply app/metrics.yaml:1628]: ping @eliserichards

**Comment by eliserichards** [comment]
The QR scanner telemetry can be removed. There will be changes made to the functionality in #16189 and I want to have another discussion with Vesta/Marissa about what events will be useful after the UX work is finalized.

**Review by eliserichards — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**